app/src/androidTest/java/com/android/partagix/home/HomeTest.kt

```diff
@@ -55,4 +98,44 @@ class HomeTest : TestCase(kaspressoBuilder = Kaspresso.Builder.withComposeSuppor
       bottomNavBarItemInventory { assertIsDisplayed() }
     }
   }
+
+  @Test
+  fun firstBigButton() = run {
+    ComposeScreen.onComposeScreen<HomeScreen>(composeTestRule) {
+      firstBigButton {
+        assertIsDisplayed()
+        performClick()
+      }
+    }
+  }
+
+  @Test
+  fun secondBigButton() = run {
+    ComposeScreen.onComposeScreen<HomeScreen>(composeTestRule) {
+      secondBigButton {
+        assertIsDisplayed()
+        performClick()
+      }
+    }
+  }
+
+  @Test
+  fun thirdBigButton() = run {
+    ComposeScreen.onComposeScreen<HomeScreen>(composeTestRule) {
+      thirdBigButton {
+        assertIsDisplayed()
+        performClick()
+      }
+    }
+  }
+
+  @Test
+  fun itemList() = run {
+    ComposeScreen.onComposeScreen<HomeScreen>(composeTestRule) {
+      itemList {
```

Comment:
You could test the scroll feature of the item list.

Reply:
The final screen should display only the 3 first items from the borrowing requests. The current version is only to wait for the viewModel